src/Microsoft.AspNet.Hosting/Program.cs

```diff
@@ -24,9 +24,9 @@ public Program(IServiceProvider serviceProvider)
         public void Main(string[] args)
         {
             var config = new Configuration();
-            if (File.Exists(HostingIniFile))
+            if (File.Exists(HostingJsonFile))
             {
-                config.AddIniFile(HostingIniFile);
+                config.AddJsonFile(HostingJsonFile);
```

Comment:
Do you need to pull in the config json dependency for this? I don't think we want to do that in Hosting as it brings in Newtonsoft.json.